_posts/how_API_works.md

```diff
@@ -0,0 +1,714 @@
+
+Blogger also has private blogs, which require authentication.
+
+Response
+If the request succeeds, the server responds with an HTTP 200 OK status code and the list of pages:
+
+
+{
+  "kind": "blogger#pageList",
+  "items": [
+    {
+      "kind": "blogger#page",
+      "id": "273541696466681878",
+      "blog": {
+        "id": "4967929378133675647"
+      },
+      "published": "2011-07-14T16:16:00.000Z",
+      "updated": "2011-07-14T16:16:23.602Z",
+      "url": "http://brettmorgan-test-blawg.blogspot.com/p/static-content.html",
+      "selfLink": "https://www.googleapis.com/blogger/v3/blogs/4967929378133675647/pages/273541696466681878",
+      "title": "Static Content",
+      "content": "elided for readability",
+      "author": {
+        "id": "901569848744",
+        "displayName": "brett",
+        "url": "http://www.blogger.com/profile/16258312240222542576",
+        "image": {
+          "url": "https://resources.blogblog.com/img/b16-rounded.gif"
+        }
+      }
+    }
+  ]
+}
+Retrieving a specific page
+You can retrieve a specific page from a blog by sending a GET request to the pages resource URI. The URI for a pages Resource has the following format:
+
+
+https://www.googleapis.com/blogger/v3/blogs/blogId/pages/pageId
+Request
+
+GET https://www.googleapis.com/blogger/v3/blogs/4967929378133675647/pages/273541696466681878?key=YOUR-API-KEY
+"A user does not need to be authenticated to retrieve a public blog. The application does not need to include Authorization HTTP header for a public blog request; however, you do need to provide the API key.
+
+Blogger also has private blogs, which require authentication.
+
+Response
+If the request succeeds, the server responds with an HTTP 200 OK status code and the page data:
+
+
+{
+  "kind": "blogger#page",
+  "id": "273541696466681878",
+  "blog": {
+    "id": "4967929378133675647"
+  },
+  "published": "2011-07-14T16:16:00.000Z",
+  "updated": "2011-07-14T16:16:23.602Z",
+  "url": "http://brettmorgan-test-blawg.blogspot.com/p/static-content.html",
+  "selfLink": "https://www.googleapis.com/blogger/v3/blogs/4967929378133675647/pages/273541696466681878",
+  "title": "Static Content",
+  "content": "elided for readability",
+  "author": {
+    "id": "901569848744",
+    "displayName": "brett",
+    "url": "http://www.blogger.com/profile/16258312240222542576",
+    "image": {
+      "url": "https://resources.blogblog.com/img/b16-rounded.gif"
+    }
+  }
+}
+Working with users
+Retrieving a user
+You can retrieve a user's information by sending an HTTP GET request to the users resource URI:
+
+
+https://www.googleapis.com/blogger/v3/users/userId
+Request
+
+GET https://www.googleapis.com/blogger/v3/users/self
+Authorization: /* OAuth 2.0 token here */
+Note: The user must be authenticated to list their own information, so you must provide the Authorization HTTP header with the GET request.
```

Comment:
Add proper markdown editing you can refer [this](https://guides.github.com/features/mastering-markdown/)